### PR TITLE
Feature/Arkera - 2112 verify suggestion is moved to do list on click of plus sign

### PR DESCRIPTION
**Pull request by sachinpkulkarni**

### CODE REVIEW

File: page_objects/sales_alert_page.py [hunk 1]
```diff
@@ -32,12 +34,38 @@ def get_thread_name(self):
...
+                # if element.find_element_by_locator(locators['title_text']):
+                suggestion_list.append(element.text)
+            except NoSuchElementException:
+                pass
+        return suggestion_list
+
+    '#Function to take action on suggestion - cancel or add'
+    def suggestion_action(self, action_locator):
+        if action_locator == 'accept_button':
+            suggestion_list = self.get_suggestion_name_list()
```
Comment (jeevanps) [inline page_objects/sales_alert_page.py]: Change the actions to be simple as accept, reject etc

File: page_objects/sales_alert_page.py [hunk 2]
```diff
@@ -32,12 +34,38 @@ def get_thread_name(self):
...
+        for element in suggestion_name:
+            try:
+                # if element.find_element_by_locator(locators['title_text']):
+                suggestion_list.append(element.text)
+            except NoSuchElementException:
+                pass
+        return suggestion_list
+
+    '#Function to take action on suggestion - cancel or add'
+    def suggestion_action(self, action_locator):
```
Comment (jeevanps) [inline page_objects/sales_alert_page.py]: change the name action_locator to some meaning full name.

File: page_objects/sales_alert_page.py [hunk 3]
```diff
@@ -32,12 +34,38 @@ def get_thread_name(self):
...
+            if suggestion_list.__len__() > 0:
+                object_for_action = self.find_element_by_locator(locators['accept_object'])
+                object_for_action.click()
+            else:
+                print 'suggestions are not available'
+        elif action_locator == 'reject_button':
+                suggestion_list = self.get_suggestion_name_list()
+                if suggestion_list.__len__() > 0:
+                    object_for_action = self.find_element_by_locator(locators['cancel_object'])
+                    object_for_action.click()
```
Comment (jeevanps) [inline page_objects/sales_alert_page.py]: I think we can combine the two lines of code

File: py_test_directory/test_suite_sales_alert_card.py [hunk 4]
```diff
@@ -68,3 +83,19 @@ def test_sales_alert_card_in_details(self):
...
+    def test_cancel_of_suggestion_in_sales_alert_card(self):
+        home_page = HomePage(self.driver)
+        home_page.click_back_button()
+        home_page.wait_until_loaded(home_page.get_object_locator("home_card_gen_date"))
+        sales_alert_details = home_page.select_sales_alert_card("sales_alert_card")
+        sales_alert_page = SalesAlertPage(self.driver)
+        sales_alert_page.wait_until_loaded(sales_alert_page.get_object_locator('sales_alert_body'))
+        suggestion_list = sales_alert_page.get_suggestion_name_list()
+        sales_alert_page.suggestion_action('cancel_object')
+        sales_alert_page.wait_until_loaded(sales_alert_page.get_object_locator('confirmation_popup'))
```
Comment (jeevanps) [inline py_test_directory/test_suite_sales_alert_card.py]: rename cancel_object in a correct way.

File: py_test_directory/test_suite_sales_alert_card.py [hunk 5]
```diff
@@ -39,6 +39,22 @@ def test_select_sales_alert_card(self):
...
+    '#Test verification of accept of suggestion on sales alert card in details screen'
+    def test_accept_of_suggestion_in_sales_alert_card(self):
+        sales_alert_page = SalesAlertPage(self.driver)
+        navigate_to_home(self.driver)
+        home_page = HomePage(self.driver)
+        home_page.wait_until_loaded(home_page.get_object_locator("home_card_gen_date"))
+        home_page.select_home_card('sales_alert_card')
+        sales_alert_page.wait_until_loaded(sales_alert_page.get_object_locator('sales_alert_body'))
+        suggestion_list = sales_alert_page.get_suggestion_name_list()
+        sales_alert_page.suggestion_action('accept_button')
```
Comment (jeevanps) [inline py_test_directory/test_suite_sales_alert_card.py]: rename accept button in the correct way

**Comment by sachinpkulkarni** [comment]
Closing this PR, as latest code has been updated in following PR

**Comment by sachinpkulkarni** [comment]
https://github.com/CarbonNV/AdvisorPortal-IntegrationTests/pull/2

**Pull request closed without merging**